### PR TITLE
tasks: release: add an invoke task to update last_stable fields

### PR DESCRIPTION
**Pull request by chouquette**
### What does this PR do?

This PR updates `release.json` last_release field when finalizing a non-RC version

### Motivation

So we don't have to update that field manually

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Comment by pgimalac** [comment]
Nice ! 
One thing I'm not sure of is whether we want to update the field when doing the release or just after ?
Imagine the merge pipeline to main fails, artifacts aren't generated / published, we need to revert the change in last stable
There could also be a race issue: between the merge and the end of the pipeline the last stable entry points to a version which is not yet available

File: tasks/release.py [hunk 1]
```diff
@@ -720,6 +720,9 @@ def update_release_json(new_version: Version, max_version: Version):
 
     release_entry = release_entry_for(new_version.major)
     print(f"Updating {release_entry} for {new_version}")
+    # If the release isn't a RC, update the last stable release field
+    if not new_version.is_rc():
```
Comment (chouetz) [inline tasks/release.py:724]: Should it be part of the `_update_release_json` method instead?

Comment (chouquette) [reply tasks/release.py:724]: Indeed that seems more sensible, I"ll update

**Comment by chouquette** [comment]
> Nice ! One thing I'm not sure of is whether we want to update the field when doing the release or just after ? Imagine the merge pipeline to main fails, artifacts aren't generated / published, we need to revert the change in last stable There could also be a race issue: between the merge and the end of the pipeline the last stable entry points to a version which is not yet available

Indeed you are correct, this is executed before the build starts so this is invalid. I'll try to find another entry point.

I'd rather have this done automatically as part of an existing task, and not have to fall back to a new task since it wouldn't really help as far as forgetting to update that field goes.

**Review by amenasria — APPROVED** [review]
LGTM !

**Pull request merged**